nominal/core.py

```diff
@@ -928,6 +932,19 @@ def get_attachments(self, rids: Iterable[str]) -> Sequence[Attachment]:
         """Retrive attachments by their RIDs."""
         return list(self._iter_get_attachments(rids))
 
+    def get_all_units(self) -> Sequence[scout_units_api.Unit]:
+        """Retrieve list of all allowable units"""
+        response = self._units_client.get_all_units(self._auth_header)
+        return [unit for units in response.units_by_property.values() for unit in units]
+
+    def get_unit(self, unit_symbol: str) -> Optional[scout_units_api.Unit]:
```

Comment:
```suggestion
    def get_unit(self, unit_symbol: str) -> scout_units_api.Unit | None:
```
nit: this library prefers `T | None` to `Optional[T]` as per the [python stdlib recommendation](https://docs.python.org/3/library/typing.html#typing.Union)
> ... the shorthand `int | str`. Using that shorthand is recommended.